subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -88,14 +95,19 @@ static void iso_rx_demux(void *param);
 #endif /* CONFIG_BT_CTLR_SYNC_ISO) || CONFIG_BT_CTLR_CONN_ISO */
 
 #if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
-static MFIFO_DEFINE(iso_tx, sizeof(struct lll_tx),
-		    CONFIG_BT_CTLR_ISO_TX_BUFFERS);
-
```

Comment:
I didn't get rid of the iso_tx mfifo, but kept the whole demuxing path as for ACL. But I can see that we don't actually need it, so great optimization!